web/modules/custom/dpl_loans/src/Plugin/Block/LoanListBlock.php

```diff
@@ -102,7 +102,6 @@ public function build() {
 
       // Urls.
       'ereolen-my-page-url' => dpl_react_apps_format_app_url($generalSettings->get('ereolen_my_page_url'), GeneralSettingsForm::EREOLEN_MY_PAGE_URL),
-      'material-overdue-url' => dpl_react_apps_format_app_url($loanListSettings->get('material_overdue_url'), DplLoansSettings::MATERIAL_OVERDUE_URL),
```

Comment:
Please reconsider this change.

We cannot simply remove the url. The React applications still need to know where to direct users who want to see their list of overdue materials (probably loans) and this is passed through this value.

To be honest I do not know what the url for this should be but to me it seems reasonable to use the [`dpl_loans.list` route](https://github.com/reload/dpl-cms/blob/main/web/modules/custom/dpl_loans/dpl_loans.routing.yml#L1-L1) for now.